contracts/interfaces/IPool.sol

```diff
@@ -154,6 +154,8 @@ interface IPool {
     */
     function validatorRegistration() external view returns (IDepositContract);
 
+    function receiveMEV() external payable;
```

Comment:
I suggest renaming it to `receiveFees`, also docstring is missing

Reply:
- [x] Done